### PR TITLE
AG-12847 - update module names

### PR DESCRIPTION
**Pull request by peterjrreynolds**

### CODE REVIEW

File: packages/ag-grid-community/src/modules/moduleRegistry.ts [hunk 1]
```diff
@@ -94,14 +93,12 @@ export function _assertModuleRegistered(
         return true;
     }
 
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    const modName = Object.entries(ModuleNames).find(([k, v]) => v === moduleName)?.[0];
-    const warningMessage = `AG Grid: unable to use ${reason} as the ${modName} is not registered${areGridScopedModules ? ` for gridId: ${gridId}` : ''}. Check if you have registered the module:
-       
+    const isEnterprise = false; // TODO - enterprise check
```
Comment (StephenCooper) [inline packages/ag-grid-community/src/modules/moduleRegistry.ts]: Use module enterprise flag

File: packages/ag-grid-community/src/syncService.ts [hunk 2]
```diff
@@ -51,7 +50,7 @@ export class SyncService extends BeanStub implements NamedBean {
         this.eventService.dispatchEvent({
             type: 'gridReady',
         });
-        const isEnterprise = this.gos.isModuleRegistered(ModuleNames.EnterpriseCoreModule);
+        const isEnterprise = this.gos.isModuleRegistered('EnterpriseCoreModule');
```
Comment (StephenCooper) [inline packages/ag-grid-community/src/syncService.ts]: Move inside of the debug

**Review by StephenCooper — APPROVED** [review]
Approved with small changes.

File: packages/ag-grid-community/src/modules/moduleRegistry.ts (hunk 1 above)
Comment (peterjrreynolds) [reply packages/ag-grid-community/src/modules/moduleRegistry.ts]: Can't do this as we only have the module name, not the module itself. Will have to have a list of enterprise module names. Need to move this into the validation service

Comment (peterjrreynolds) [reply packages/ag-grid-community/src/modules/moduleRegistry.ts]: Will do this as a separate task

Comment (peterjrreynolds) [reply packages/ag-grid-community/src/modules/moduleRegistry.ts]: Done as part of this

File: packages/ag-grid-community/src/syncService.ts (hunk 2 above)
Comment (peterjrreynolds) [reply packages/ag-grid-community/src/syncService.ts]: Done

**Pull request merged**